src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -641,6 +641,17 @@ public boolean verifyNewRankRecipientsQuestionFormIsDisplayed() {
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
 
+    /*
+     * checks if alert class is enabled on the visibility options div for the specified question number.
+     * To be used only with questions with responses.
+     */
+    public boolean verifyAlertClassIsEnabledForVisibilityOptions(int questionNo) {
```

Comment:
Rename to `isAlertClassEnabledForVisibilityOptions` to follow our boolean naming conventions. Update the javadoc by deleting the second sentence and capitalising the first sentence.